hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/rollback/TestCopyOnWriteRollbackActionExecutor.java

```diff
@@ -431,7 +433,7 @@ public void testRollbackForMultiwriter() throws Exception {
   public void testRollbackWhenReplaceCommitIsPresent() throws Exception {
 
     // insert data
-    HoodieWriteConfig writeConfig = getConfigBuilder().withAutoCommit(false).build();
+    HoodieWriteConfig writeConfig = getConfigBuilder().withAutoCommit(false).withEmbeddedTimelineServerEnabled(false).build();
```

Comment:
Avoid `.withEmbeddedTimelineServerEnabled(false)`; if test fails because of this, then the production code need to be fixed.

Reply:
+1

Reply:
I have removed the changes based on comment - https://github.com/apache/hudi/pull/13007#discussion_r2017964552